games-util/Steam-appimage/Steam-appimage-1.0.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit wrapper desktop
+
+DESCRIPTION="Unofficial Steam AppImage built on top of Conty (Arch Linux), with deps inside"
+HOMEPAGE="https://github.com/ivan-hc/Steam-appimage"
+SRC_URI="https://github.com/ivan-hc/${PN}/releases/download/continuous/Steam-${PV}.79-2-3-x86_64.AppImage"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="strip"
+
+RDEPEND="sys-fs/fuse:0"
+
+QA_PREBUILT="*"
+
+src_install() {
+	local file=Steam-${PV}.79-2-3-x86_64.AppImage
+
+	mkdir -p "${ED}/opt/"
+	cp "${DISTDIR}/$file" "${ED}/opt/Steam.AppImage" || die
+	fperms +x /opt/Steam.AppImage
+
+	make_wrapper ${PN} "env FUSERMOUNT_PROG=\$(which fusermount) /opt/Steam.AppImage"
```

Comment:
It feels surprising to me that appimage wouldn't internally check PATH for `fusermount`, can someone verify this?

Either way you cannot use the `which` program for it. Simply use `command -v` in all cases of shell programs where you might think to use `which`.

Note that if upstream software already uses `which` and you don't apply a patch to fix this you must add a new RDEPEND for the package that provides `which`.

Reply:
See e.g.
https://unix.stackexchange.com/a/85250

Reply:
Related issue https://github.com/ivan-hc/Steam-appimage/issues/5

Reply:
@eli-schwartz 

>It feels surprising to me that appimage wouldn't internally check PATH for fusermount, can someone verify this?

I noticed you already checked the issue, the big surprise here is that even though Gentoo has `fusermount` in `/usr/bin` the static runtime doesn't pick it because of something to do with read permissions which I'm not very sure why it happens. 

@vitaly-zdanevich you were told to change `$(which fusermount)` for `$(command -v fusermount)`, which is correct since command -v is POSIX.

I actually didn't know you put the fix in the code, I was talking about the bug in this PR thinking it wasn't there.

Reply:
Replaced to `command -v`